src/components/Order/OrderItems.vue

```diff
@@ -49,28 +49,20 @@ export default {
     methods:{
 
         getThePrducts(){
-            axios.get(`${this.baseURL}order/?token=${this.token}`).then((response) => {
-                if(response.status === 200) { 
-                    this.products = response.data
-                    let len = Object.keys(this.products).length
-                    let i,j
-                    let paramId=parseInt(this.orderID)
-                    for(i=0;i<len;i++){
-                        let id=this.products[i].id 
-                        if(id === paramId){ 
-                            break
-                        }
-                    }
-                    this.lengthofOrderItems = Object.keys(this.products[i].orderItems).length
-                    for(j=0;j<this.lengthofOrderItems;j++){
-                        this.orderProducts.push({
-                            imgUrl:this.products[i].orderItems[j].product.imageURL,
-                            pName:this.products[i].orderItems[j].product.name,
-                            pDescription:this.products[i].orderItems[j].product.description,
-                            pPrice:this.products[i].orderItems[j].product.price,
-                            pQuantity:this.products[i].orderItems[j].quantity
-                        })
-                    }
+          axios.get(`${this.baseURL}order/${this.orderID}?token=${this.token}`).then((response) => {
+                if(response.status === 200) {
+                  let j;
+                  this.product = response.data
+                  this.lengthofOrderItems = Object.keys(this.product.orderItems).length
+                  for(j=0;j<this.lengthofOrderItems;j++){
+                    this.orderProducts.push({
```

Comment:
this.lengthofOrderItems.forEach((item)=> {
// process item
});

Reply:
Done